subsys/bluetooth/mesh/prov_bearer.h

```diff
@@ -104,8 +104,8 @@ struct prov_bearer {
 	void (*link_close)(enum prov_bearer_link_status status);
 };
 
-extern const struct prov_bearer pb_adv;
-extern const struct prov_bearer pb_gatt;
+extern const struct prov_bearer bt_mesh_pb_adv;
+extern const struct prov_bearer bt_mesh_pb_gatt;
 
 void pb_adv_init(void);
```

Comment:
Should this API also be updated?

Reply:
do later...

Reply:
In principle, all structures, macros, enums, and functions that are not declared static need to be modified? But is this necessary?

Reply:
It seems that it is not limited to Bluetooth Mesh, or it should rise to all subsystem.

Reply:
If I see the problem correctly, I think adding prefix for variables and functions is enough. Unless explicitly included structs, macros, enums won't make problems.